client.go

```diff
@@ -110,6 +119,80 @@ func chopContentType(contentType string) string {
 	return strings.Split(contentType, ";")[0]
 }
 
+// Subscribe function subscribes for events form the StreamClient TopicName after the given offset. An offset of zero should be
```

Comment:
This sentence reads weird.

Reply:
I revised it a bit. Please feel free to suggest an alternative if it still seems off.